sycl/include/CL/sycl/detail/scheduler/commands.hpp

```diff
@@ -290,6 +297,22 @@ class ExecCGCommand : public Command {
   std::unique_ptr<detail::CG> MCommandGroup;
 };
 
+// The main purpose of this class is to update host requirements
```

Comment:
If we change `UpdateRequirementCommand ` to something like `UpdateHostRequirementCommand ` this comment won't be needed.